README.md

```diff
@@ -44,7 +44,7 @@ Execute the following commands to install the tool:
 
 ```console
 git clone https://github.com/bitnami-labs/readme-generator-for-helm
-npm install -g readme-generator-for-helm
+sudo npm install -g ./readme-generator-for-helm
```

Comment:
you should not tinker with the permissions of those directories.
If you use `-g` you have to do it with `sudo`

Reply:
The `-g` option install the packages into the configured prefix. Check [this](https://docs.npmjs.com/cli/v8/configuring-npm/folders#prefix-configuration).

Taking that into account, when installing node, if it has been installed into a "global" system path, i.e. for all the users, for example, it is typical to install it under `/usr/local` it will require `sudo`.
If the system has more users, it would be a good practise that each user will have its own node installation, they could want to use different versions, etc. In that case, let's say you install it under `${HOME}/bin`. The prefix in that case won't need root permissions.

So, in short, I think this shouldn't be added by default. Each user should know how its system is configured, and use `sudo` if the certain case requires it. But setting it by default will be telling to the users to use `sudo` always, which depending on your node installation path could or could not be necessary.

Reply:
@rafariossaa Is there any consensus?

Reply:
We agreed in changing the instructions and adding a small comment for the user.
Depending on how the user has installed node he might need to adjust the `PATH`.